spec/features/orders_view_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+describe "the order view", type: :feature do
+
+  before :each do
+    @admin = FactoryGirl.create(:admin)
+    @order = FactoryGirl.create(:order)
+    visit login_path
+    fill_in("session_email", with: @admin.email)
+    fill_in("session_password", with: @admin.password)
+    click_button 'Sign in'
+    visit orders_path
+  end
+
+  it "displays all the orders on the index page" do
+    expect(page).to have_content("Orders")
+  end
+
+  it "displays links for each individual order" do
+    expect(page).to have_content("click here")
+  end
+
+  it "shows each orders indivdual information" do
+    click_link_or_button( 'click here')
```

Comment:
Space inside parentheses detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.